ib_insync/util.py

```diff
@@ -511,7 +511,7 @@ def formatIBDatetime(dt: Union[date, datetime, str, None]) -> str:
     return s
 
 
-def parseIBDatetime(s: str):
+def parseIBDatetime(s: str) -> datetime.datetime:
```

Comment:
The first branch in here can return a `date` instead of a `datetime`, so you might want:
```suggestion
def parseIBDatetime(s: str) -> Union[datetime.date, datetime.datetime]:
```

Reply:
oops. you are right of course.